build/js/main-compiled.js

```diff
@@ -0,0 +1,208 @@
+'use strict';
+
+// import smoothScroll from 'smoothScroll';
```

Comment:
@PalleZingmark I can't seem to get import to work - can you see what I am doing wrong?